### PR TITLE
238 array field

### PR DESCRIPTION
**Pull request by micahchiang**
## Description

This pull request adds support for array fields in VAFSC

## Original issue(s)

department-of-veterans-affairs/va-forms-system-core#238

## Definition of done

- [ ] Package.json version has been updated to match Github release tag
- [ ] Documentation has been updated, if applicable
- [x] A link to the original github issue has been provided
- [x] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,9 +104,9 @@
     "jest": "^27.0.4",
     "lint-staged": "^11.0.0",
     "prettier": "^2.3.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-is": "^17.0.2",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-is": "^17.0.0",
```
Comment (nickjg231) [inline package.json:109]: These packages were downgraded for the time being because of a dependency issue with the `<FieldArray>` element

**Comment by nickjg231** [comment]
Some of our React library packages were downgraded for the time being because of a dependency issue with the `<FieldArray>` element:
image

**Comment by mlauren** [comment]
> Some of our React library packages were downgraded for the time being because of a dependency issue with the `<FieldArray>` element: image

@nickjg231 would double check with @bjmccotter7192 that this isn't going to cause any issues when pulled into vets website

**Comment by micahchiang** [comment]
> > > Some of our React library packages were downgraded for the time being because of a dependency issue with the `<FieldArray>` element: image
> > 
> > 
> > @nickjg231 would double check with @bjmccotter7192 that this isn't going to cause any issues when pulled into vets website
> 
> So in vets-website they use `17.0.2` on those packages so is there a way to match the types to `17.0.2`?

This might not actually resolve the FieldArray build error, so this change may be removed from the PR. Doing a little more research into it.

**Pull request closed without merging**